about.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js" defer></script>
+</head>
+<body>
+    <section class="header-section">
+        <div class="main-containers">
+            <nav class="inner-header">
+                <li class="inner-list"><img src="images/facebook.png" alt=""></li>
+                <li class="inner-list"><img src="images/twitter.png" alt=""></li>
+                <li class="inner-list"><a href="#">English</a></li>
+                <li class="inner-list"><a href="#">My page</a></li>
+                <li class="inner-list"><a href="#">Logout</a></li>
+            </nav>
+        </div>
+            <header>
+                <div class="main-containers">
+                    <div class="header-container">
+                      <nav class="header">
+                        <div class="my-logo">
+                            <a href="#"><img class="logo" src="images/Ifeanyi_Logo_1B-01-removebg-preview.png" alt=""></a>
+                        </div>
+                        <div class="hambuger">
+                          <div class="bar"></div>
+                          <div class="bar"></div>
+                          <div class="bar"></div>
+                        </div>
+                        <ul class="nav-list">
+                          <li class="nav-item about"><a href="#">About</a></li>
+                          <li class="nav-item"><a href="#section1">Program</a></li>
+                          <li class="nav-item"><a href="#section2">join</a></li>
+                          <li class="nav-item"><a href="#section3">Sponsor</a></li>
+                          <li class="nav-item"><a href="#">News</a></li>
+                          <li class="nav-item"><a class="campaign" href="#">Campaign</a></li>
+                        </ul>
```

Comment:
- [ ] Kindly add a link to the home page as this is a requirement, that the navigation should have a way to move between pages.
![image](https://user-images.githubusercontent.com/81932478/196728356-bb0de42e-6c24-4c69-bff0-4b3b5458452a.png)

Reply:
Hi, @Munsa1 Thanks for the feedback,
I'll add the link to the homepage